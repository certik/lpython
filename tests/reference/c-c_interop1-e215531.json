{
    "basename": "c-c_interop1-e215531",
    "cmd": "lpython --no-color --show-c {infile}",
    "infile": "tests/c_interop1.py",
    "infile_hash": "e676d05044fa71c2bd65b09dd5ed2d20414f6b1c028ef528aab47748",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "c-c_interop1-e215531.stdout",
    "stdout_hash": "ac7de87134ed10075c53ad60385310b20178165421d49aa124020905",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}