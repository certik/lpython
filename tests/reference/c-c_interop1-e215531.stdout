#include <assert.h>
#include <complex.h>
#include <inttypes.h>
#include <stdio.h>
#include <math.h>

#include <lfortran_intrinsics.h>

double f(double x);

void g(double a, float b, int64_t c, int32_t d);

double h(double x)
{
    double _lpython_return_variable;
    _lpython_return_variable = x + 1.000000;
    return _lpython_return_variable;
}

void l(double a, float b, int64_t c, int32_t d)
{
    printf("%s\n", "OK");
}

void main0()
{
    double i;
    double x;
    float y;
    int64_t z;
    int32_t zz;
    x = 5.000000;
    i = f(x);
    y = 5.400000;
    z = 3;
    zz = 2;
    g(x, y, z, zz);
    i = h(x);
    l(x, y, z, zz);
}

int main(int argc, char* argv[])
{
    return 0;
}
